### DISCUSSION TITLE
API: Use existing user and event resources for `/notifications` endpoint

### DISCUSSION THREAD

**Discussion by marhei** [Traewelling/traewelling#2716] [Ideas / Feature Requests]
Good morning,

The `/notifications` endpoint is currently not really documented. To improve this situation, I would suggest using the already existing `LightUserResource` and `EventResource` where a user or event is provided as notification data, as currently only a special resource without any documentation is provided in these cases.

I would also like to hear @jheubuch's thoughts on this approach.

**Comment by jheubuch** [comment]
I like the idea of having consistent models. And it would be nice if we could have this here, too.

Nevertheless: If this is changed, the old and new models should be both delivered for a couple of transitional period, that our applications don't break.
